### DISCUSSION TITLE
How make filter on secondaryheader

### DISCUSSION THREAD

**Discussion by ramadiiw** [rappasoft/laravel-livewire-tables#1658] [Q&A] [answered]
hello everyone i was just trying to create a datatable using this. I applied this with Bootstrap5, in the demo there is a select filter in the secondary header. I've been trying to find a way to do this, but haven't found any examples.

**Comment by lrljoe** [comment, marked as answer]
There's full documentation on the Rappasoft site:
[Select Filter](https://rappasoft.com/docs/laravel-livewire-tables/v3/filter-types/filters-select)

Then an Example is here:
[Example](https://rappasoft.com/docs/laravel-livewire-tables/v3/examples/advanced-example)

**Reply by lrljoe** [reply]
Just tested in my development branch, and works okay.  Sharing your full Table component code if it **isn't** working will help identify the missing piece :)

**Reply by ramadiiw** [reply]
Thank you for your help, I just found the way.

**Reply by lrljoe** [reply]
Great news, the Discord community is typically more active than on here, so do feel free to reach out with anything further on there :)
